sdk-cpp/tests/download_properties_tests.cpp

```diff
@@ -51,23 +42,35 @@ class DownloadPropertyTestsDOSVC_NoThrow : public ::testing::Test
         auto ec = op();
         VerifyError(ec.value(), expectedErrors);
     }
+#endif
 };
 
+// Only the DoSvc client (windows) supports get/set property at present
```

Comment:
what's TimeOperation have to do with get/set property?

Reply:
Comment is meant for the ifdef. See the last test case here - TimeOperation is used to (indirectly) verify that the foreground/background property has taken effect by comparing the time taken to download.